src/protocol/requests/deleteRecords/v0/response.spec.js

```diff
@@ -0,0 +1,25 @@
+const { decode, parse } = require('./response')
+
+// FIXME - update with correct sample response
```

Comment:
Is this comment still relevant?

Reply:
D'oh, apologies, indeed, I fixed the test but forgot to remove the fixme.